sdcm/nemesis.py

```diff
@@ -3970,8 +3977,8 @@ def disrupt_run_cdcstressor_tool(self):
         ks_tables_with_cdc = self.cluster.get_all_tables_with_cdc(self.target_node)
         if not ks_tables_with_cdc:
             self.log.warning("CDC is not enabled on any table. Skipping")
-            UnsupportedNemesis("CDC is not enabled on any table. Skipping")
-            return
+            raise UnsupportedNemesis("CDC is not enabled on any table. Skipping")
+
```

Comment:
It looks like a totally unrelated to the PR idea.
This fix must be in a separate PR.